README.md

```diff
@@ -1,24 +1,89 @@
-# README
+# Reserve Classes - Back-end
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The project we built for the Final Capstone Project is based on an app to book an appointment to attend a class. We had to follow the given design of the website, but we personalize the content, i.e., instead of booking an appointment to try a motorcycle, we built an app to book an appointment to attend a class.
 
-Things you may want to cover:
+### Entity Relationship Diagram (ERD)
+![](/images/database.png)
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- PostgreSQL
 
-* Configuration
+## Kanban board
+- Click [here](https://github.com/christianbiring1/reserve_classes_back-end/projects/1) to view the Kanban board
+- Click [here](https://github.com/christianbiring1/reserve_classes_back-end/issues/7) to view the Kanban board issue
 
-* Database creation
+## Front end
+> Click [here](https://github.com/christianbiring1/reserve_classes_front-end) to view the front end repository.
 
-* Database initialization
+## Live Demo
 
-* How to run the test suite
+Click [here](https://calm-mesa-77347.herokuapp.com/) to follow the API documentation
 
-* Services (job queues, cache servers, search engines, etc.)
+Not yet available
 
-* Deployment instructions
+### Prerequisites
 
-* ...
+- To run this project in your local environment, You will need to have the following tools installed loally :
+
+- `Ruby 3.0+`
+- `Rails 6.0+`
+- `PostgreSQL` version 14+
+
+## Set up
+
+- `git clone git@github.com:christianbiring1/reserve_classes_back-end.git`
+- `cd reserve_classes_back-end`
+- `bundle install`
+- `rails db:create`
+- `rails db:migrate`
+if you want to run it locally you will need to change the URL host in the `initializer/cors`and change it to localhost:3000 the domain of the font-end though. And all the API end point on the front-end app must match that as well.
```

Comment:
Maybe consider adding something on running tests here as well.

Reply:
Yeah sure.
Thank you @mcrd25 :+1: